Mathlib/Tactic/Linter/Style.lean

```diff
@@ -68,4 +69,41 @@ initialize addLinter setOptionLinter
 
 end Style.setOption
 
+/-- The `check_declID` linter emits a warning on declarations whose name is non-standard style.
+(Currently, this only includes declarations whose names include a double underscore.)
+
+**Why is this bad?** Double underscores in theorem names can be considered non-standard style and
+probably have been introduced by accident
+**How to fix this?** Use single underscores to separate parts of a name, following standard naming
+conventions.
+-/
+register_option linter.style.check_declID : Bool := {
```

Comment:
How about calling this
* `linter.style.doubleUnderscore` and leave a rename to a possible extension that checks more;
* `linter.style.nameCheck`, so we avoid the "technical" `declId` and make this more memorable?

Reply:
renamed in that spirit 
Now we have 
- Linter.linter.style.nameCheck  
- Linter.Style.nameCheck.doubleUnderscore